test/helpers/node.go

```diff
@@ -47,6 +47,14 @@ func (s *SSHMeta) String() string {
 
 }
 
+func GetLoSSHMeta() *SSHMeta {
+	return &SSHMeta{
+		sshClient: GetSSHClient("127.0.0.1", 22, "john"),
```

Comment:
Could probably parameterize the user option here ;)

Reply:
Or everyone can use "john" as their username... will fix along with whatever is causing an error on my system now.